commit a6e0d179764cb31b2981c85e6fd156adc777e4ed
Author: Dmitry Bogdanov <d.bogdanov@yadro.com>
Date:   Mon May 23 12:59:05 2022 +0300

    scsi: target: iscsi: Control authentication per ACL
    
    Add acls/{ACL}/attrib/authentication attribute that controls authentication
    for particular ACL. By default, this attribute inherits a value of the
    authentication attribute of the target port group to keep backward
    compatibility.
    
    Authentication attribute has 3 states:
    
     "0" - authentication is turned off for this ACL
    
     "1" - authentication is required for this ACL
    
     "-1" - authentication is inherited from TPG
    
    Link: https://lore.kernel.org/r/20220523095905.26070-4-d.bogdanov@yadro.com
    Reviewed-by: Roman Bolshakov <r.bolshakov@yadro.com>
    Reviewed-by: Konstantin Shelekhin <k.shelekhin@yadro.com>
    Reviewed-by: Mike Christie <michael.christie@oracle.com>
    Signed-off-by: Dmitry Bogdanov <d.bogdanov@yadro.com>
    Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>

diff --git a/drivers/target/iscsi/iscsi_target_nodeattrib.c b/drivers/target/iscsi/iscsi_target_nodeattrib.c
index 874cb33c9be0..d63efdefb18e 100644
--- a/drivers/target/iscsi/iscsi_target_nodeattrib.c
+++ b/drivers/target/iscsi/iscsi_target_nodeattrib.c
@@ -30,6 +30,7 @@ void iscsit_set_default_node_attribues(
 {
 	struct iscsi_node_attrib *a = &acl->node_attrib;
 
+	a->authentication = NA_AUTHENTICATION_INHERITED;
 	a->dataout_timeout = NA_DATAOUT_TIMEOUT;
 	a->dataout_timeout_retries = NA_DATAOUT_TIMEOUT_RETRIES;
 	a->nopin_timeout = NA_NOPIN_TIMEOUT;